feed/tests/test_feed_views.py

```diff
@@ -131,3 +160,60 @@ def test_redirect_after_successful_post_creation(self, users, client):
         # The new post ID should be as the amount of posts exists
         new_post_id = Post.posts.count()
         assert response.url == POST_DETAIL_URL + str(new_post_id) + '/'
+
+
+class TestLikeForNonLoggedUser:
+    def test_like_view_return_warning_when_logged_out(self, post, client):
+        # Testing that when logged out, if trying to access the like url
+        # a warning message returns
+        post_to_like = post
+        response = client.get(post_like_url(post_to_like.id))
+        messages = [m.message for m in get_messages(response.wsgi_request)]
```

Comment:
Did you try this ?
esponse.context['messages'] instead?

Reply:
I did but with no success trying to get the message content or details.
What I get from `response.context['messages']` is `<FallbackStorage: request=<WSGIRequest: GET '/login/'>>`